### PR TITLE
`<Button/>`- migrating button UniDriver, adding "hasSkin" function and migrating tests

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary
1. Migrating button UniDriver.
2. Adding `hasSkin` function to the testkit.
3. Adding `constants` file.
4. Migrating and refactoring tests (both visual tests and unit tests). 
5. Updating `skins.md` file with the new skins added. 

Resolves  https://github.com/wix/wix-style-react/issues/3421

### ✅ Checklist
- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by TalZa1810** [comment]
Hi @mosheKerbel ,
I noticed few things: 

The implementation of the focus styling is in the [`<ButtonNext/>` component](https://github.com/wix/wix-ui/blob/master/packages/wix-ui-core/src/components/button-next/button-next.st.css#L8) (`wix-ui-core`). 

It is imported to `wix-style-react` `<Button/>` themes [here](https://github.com/wix/wix-style-react/blob/master/src/Button/Button.st.css#L7). 


Therefore, I'm not sure whether I'm suppose to test it here or not. Maybe that's why the `isFocused` function is not documented in [master](https://github.com/wix/wix-style-react/blob/master/src/Button/docs/testkit.md)? 

Should I test it in `wix-style-react` and add it to the documentation or not? WDYT?

File: src/Button/tests/Button.spec.js [hunk 1]
```diff
@@ -29,18 +44,146 @@ describe('Button', () => {
...
+      it.each(themes)(
+        'should return true when button has skin %s',
+        async theme => {
+          const props = {
+            skin: theme,
+          };
+          const { driver } = await render(
+            createButton({ ...defaultProps, ...props }),
+          );
+          expect(await driver.hasTheme(theme)).toEqual(true);
```
Comment (mosheKerbel) [inline src/Button/tests/Button.spec.js]: when i test booleans i add at least one negative scneario, to make sure that method is not always returning the same value.

**Comment by mosheKerbel** [comment]
> Hi @mosheKerbel ,
> I noticed few things:
> 
> The implementation of the focus styling is in the [`<ButtonNext/>` component](https://github.com/wix/wix-ui/blob/master/packages/wix-ui-core/src/components/button-next/button-next.st.css#L8) (`wix-ui-core`).
> 
> It is imported to `wix-style-react` `<Button/>` themes [here](https://github.com/wix/wix-style-react/blob/master/src/Button/Button.st.css#L7).
> 
> Therefore, I'm not sure whether I'm suppose to test it here or not. Maybe that's why the `isFocused` function is not documented in [master](https://github.com/wix/wix-style-react/blob/master/src/Button/docs/testkit.md)?
> 
> Should I test it in `wix-style-react` and add it to the documentation or not? WDYT?

amm let's disucss it i don't really understand what you're saying
once the `isFocused` is exported in the driver we need to verify it work by testing it....

**Comment by TalZa1810** [comment]
@mosheKerbel 
I've finished with the changes. Waiting for the CI to finish. Let me know if you think we can merge :)

**Comment by mosheKerbel** [comment]
1.failed due to 1 unrleated async timeout test
2. one applitools diff in dropdown  even though did not touch this area - let's see what happen after merge.

**Pull request merged**